docs/components/base/_index.md

```diff
@@ -152,9 +152,9 @@ Turn the base in place, rotating it to the given angle (degrees) at the given an
 **Parameters:**
 
 - `angle` [(float)](https://docs.python.org/3/library/functions.html#float): The angle to spin in degrees.
-Negative implies backwards.
+Positive implies forwards.
 - `velocity` [(float)](https://docs.python.org/3/library/functions.html#float): The angular velocity at which to spin in degrees per second.
-Negative implies backwards.
+Positive implies forwards.
```

Comment:
Same as above... what is "forward" in terms of spin?

Reply:
```suggestion
Positive implies turning to the left.
```
Ahhh I see you could spin around either way you want to to reach a given position.